package/MDAnalysis/coordinates/PDB.py

```diff
@@ -217,6 +216,17 @@ class PDBReader(base.ReaderBase):
     79 - 80        LString(2)    charge       Charge  on the atom.
     =============  ============  ===========  =============================================
 
+    Notes
+    -----
+    If a system does not have unit cell parameters (such as in electron
+    microscopy structures), the PDB file format requires the CRYST1_ field to
+    be provided with unitary values (cubic box with sides of 1 Å) and an
+    appropriate REMARK. If unitary values are found within the CRYST1_ field,
+    :code:`PDBReader` will not set unit cell dimensions and it will warn the
```

Comment:
I know this is going to get changed with #2698, but is it worth temporarily detailing that this means dimensions will return [0,0,0,0,0,0]? (i.e. "will not set unit cell dimensions (leaving it at the default [0,0,0,0,0,0[ value)...".

My thought here is mainly "if we have a big gap between this PR and the next" (e.g. a release), then more details is better.

Reply:
Sure. I think I already changed it elsewhere but obviously missed a few spots.